### DISCUSSION TITLE
byte-buddy compile dependency in jackson-databind 2.17.0

### DISCUSSION THREAD

**Discussion by uffeweil** [FasterXML/jackson-databind#4498] [General]
Hi, 

In 2.17.0 a new compile dependency has been added to byte-buddy. From the comments in the pom it seems like this should be a test dependency. 

Or is it something that should be used in "production"?

**Comment by JooHyukKim** [comment]
Hello, It's fixed in 2.17.1 via https://github.com/FasterXML/jackson-databind/pull/4429
